C:\hudson10.6.198.88-8088\workspace\copy_test>exit 1 
FATAL: command execution failed
hudson.util.IOException2: Failed to join the process
	at hudson.Proc$RemoteProc.join(Proc.java:372)
	at hudson.Launcher$ProcStarter.join(Launcher.java:283)
	at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:89)
	at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:60)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:22)
	at hudson.model.AbstractBuild$AbstractRunner.perform(AbstractBuild.java:644)
	at hudson.model.Build$RunnerImpl.build(Build.java:251)
	at hudson.model.Build$RunnerImpl.doRun(Build.java:179)
	at hudson.model.AbstractBuild$AbstractRunner.run(AbstractBuild.java:436)
	at hudson.model.Run.run(Run.java:1380)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:56)
	at hudson.model.ResourceController.execute(ResourceController.java:91)
	at hudson.model.Executor.run(Executor.java:164)
Caused by: java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at hudson.remoting.Channel$2.adapt(Channel.java:684)
	at hudson.remoting.Channel$2.adapt(Channel.java:679)
	at hudson.remoting.FutureAdapter.get(FutureAdapter.java:55)
	at hudson.Proc$RemoteProc.join(Proc.java:364)
	... 12 more
Caused by: java.lang.NullPointerException
	at hudson.Proc$LocalProc.join(Proc.java:231)
	at hudson.Launcher$RemoteLaunchCallable.call(Launcher.java:805)
	at hudson.Launcher$RemoteLaunchCallable.call(Launcher.java:775)
	at hudson.remoting.UserRequest.perform(UserRequest.java:114)
	at hudson.remoting.UserRequest.perform(UserRequest.java:48)
	at hudson.remoting.Request$2.run(Request.java:270)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at hudson.remoting.Engine$1$1.run(Engine.java:60)
	at java.lang.Thread.run(Unknown Source)
[DEBUG] Skipping watched dependency update for build: copy_test #1 due to result: FAILURE
Finished: FAILURE